index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tic Tac Toe</title>
+<link rel="stylesheet" type="text/css" href="style.css"></head>
+<body>
+<h1>Tic Tac Toe</h1>
+<div id="description"> <p>By Audrey Andoy</p>
+<p>Click a space below to begin playing! Have fun :) </p>
+</div>
+<div id="game">
+<div class="board">
+    <div class="row">
+        <div id ="a" class="space" value = " " onclick="javascript:this.style.background = '#93335d'; clickSpace('a')"></div>
```

Comment:
Here, you want to be careful about putting spaces inside of tags where they aren't expected. While Javascript can be forgiving about spaces, HTML documents can yield some unexpected results if there are extra spaces in between characters.